utilities/xmlrpcpp/libb64/src/cdecode.c

```diff
@@ -39,7 +41,7 @@ int base64_decode_block(const char* code_in, const int length_in, char* plaintex
 				if (codechar == code_in+length_in)
 				{
 					state_in->step = step_a;
-					state_in->plainchar = *plainchar;
+					state_in->plainchar = 0; // no state to save; use default value
```

Comment:
I don't know enough about base64 to understand this change, but I assume it's motivated by something which one of the new tests uncovered?

Reply:
Yes; this was uncovered by running these tests in valgrind and prevents an out-of-bounds array access through `plainchar`.

Reply:
:+1: